conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md

```diff
@@ -44,28 +44,32 @@ in place by a project (for reports made to a project) or the CPC (in the
 case of reports relating to spaces within the scope of the foundation versus a project).
 In exceptional cases where a reporter wishes to challenge the response from the CPC or
 project, or does not feel comfortable reporting through the standard processes, an escalation
-report can be made to coc-escalation@openjsf.org.
+report can be made to report@openjsf.org.
 
 All escalations will be reviewed and investigated by the Code of Conduct Panel (CoCP)
 and will result in a response that is deemed necessary and appropriate to the circumstances.
 The Code of Conduct panel is obligated to maintain confidentiality with regard
 to the reporter of an escalation.
 
-### Code of Conduct panel
+### Code of Conduct Panel
 
-The Code of Conduct panel is a cross-foundation team who have been trained in moderation
+The Code of Conduct Panel is a cross-foundation team who have been trained in moderation
 and code of conduct issues. The makeup of this team is as follows:
 
-* The Foundation Executive director
-* 1 member from the Board
-* 1 member from the CPC
-* 1 member of the OpenJS marketing committee
-* each top-level project may optionally provide 1 member
-* the non top-level projects may optionally provide 1 member
-* optionally 1 invited outside expert as agreed by the other CoCP members.
+* OpenJS Foundation Executive Director - Robin Ginn (@rginn)
+* OpenJS Foundation Marketing Lead - Kylie Wagar-Dirks (@kyliewd)
```

Comment:
```suggestion
* OpenJS Foundation Marketing representative - Kylie Wagar-Dirks (@kyliewd)
```

Reply:
I think we should explain _why_ the marketing lead is involved (because of events). Otherwise, this feels like it is about PR and managing public perception (which I know it isn't, but still).